qt/scientific_interfaces/Indirect/Reduction/ISISEnergyTransferPresenter.cpp

```diff
@@ -68,23 +68,23 @@ bool IETPresenter::validateInstrumentDetails() {
 }
 
 InstrumentData IETPresenter::getInstrumentData() {
-  auto instrument = getInstrumentName().toStdString();
-  auto analyser = getAnalyserName().toStdString();
-  auto reflection = getReflectionName().toStdString();
-
   QMap<QString, QString> instrumentDetails = getInstrumentDetails();
 
-  auto specMin = instrumentDetails["spectra-min"].toInt();
-  auto specMax = instrumentDetails["spectra-max"].toInt();
-  auto eFixed = instrumentDetails["Efixed"].toDouble();
-  auto rebinDefault = instrumentDetails["rebin-default"].toStdString();
-  auto cm1ConvertChoice = instrumentDetails["cm-1-convert-choice"] == "true";
-  auto saveNexusChoice = instrumentDetails["save-nexus-choice"] == "true";
-  auto saveAsciiChoice = instrumentDetails["save-ascii-choice"] == "true";
-  auto foldFramesChoice = instrumentDetails["fold-frames-choice"] == "true";
-
-  return InstrumentData(instrument, analyser, reflection, specMin, specMax, eFixed, rebinDefault, cm1ConvertChoice,
-                        saveNexusChoice, saveAsciiChoice, foldFramesChoice);
+  auto specMin = ;
+  auto specMax = ;
+  auto eFixed = ;
+  auto rebinDefault = ;
+  auto cm1ConvertChoice = == "true";
+  auto saveNexusChoice = ;
+  auto saveAsciiChoice = ;
+  auto foldFramesChoice = ;
```

Comment:
These wont compile and I dont think they are used either.

Reply:
Added by mistake